cmake/ChooseBlas.cmake

```diff
@@ -23,14 +23,6 @@ set(INTEL_OPT_ROOT "/opt/intel" CACHE PATH "Folder contains root-installed intel
 
 if(DEFINED USE_BLAS)
   set(BLAS "${USE_BLAS}")
-else()
-  if (USE_MKL_IF_AVAILABLE)
-    # Setting up BLAS_mkl_MKLROOT for non-Ubuntu 20.04 OSes
-    find_path(BLAS_mkl_MKLROOT mkl PATHS $ENV{MKLROOT} ${INTEL_HOME_ROOT} ${INTEL_OPT_ROOT})
-    if(NOT BLAS_mkl_MKLROOT STREQUAL "BLAS_mkl_MKLROOT-NOTFOUND")
-      set(BLAS "MKL")
-    endif()
-  endif()
```

Comment:
If the user doesn't specify any BLAS and USE_MKL_IF_AVAILABLE=ON the 1st priority is MKL, if not available try OpenBLAS and the other that is available. Can you verify that after removing those lines it is still has the same order priority?

Reply:
Well now if the user wants to use mkl he has to explicitly declare USE_BLAS=mkl. Otherwise with no BLAS set, OpenBLAS will be used. Do we want to keep this functionality now as default in mxnet?

Reply:
Using MKL for BLAS should stay as default options as it gives big performance improvements.

Reply:
Already done :)